diff --git a/meilisearch-http/Cargo.toml b/meilisearch-http/Cargo.toml
index 64766bc..d339957 100644
--- a/meilisearch-http/Cargo.toml
+++ b/meilisearch-http/Cargo.toml
@@ -97,7 +97,7 @@ mini-dashboard = [
     "zip",
 ]
 analytics = ["segment"]
-default = ["analytics", "mini-dashboard"]
+default = []

 [target.'cfg(target_os = "linux")'.dependencies]
 tikv-jemallocator = "0.4.1"
diff --git a/meilisearch-http/tests/dashboard/mod.rs b/meilisearch-http/tests/dashboard/mod.rs
index d097cfd..f5ae61e 100644
--- a/meilisearch-http/tests/dashboard/mod.rs
+++ b/meilisearch-http/tests/dashboard/mod.rs
@@ -1,24 +1,4 @@
-use crate::common::Server;
-
 #[actix_rt::test]
 async fn dashboard_assets_load() {
-    let server = Server::new().await;
-
-    mod generated {
-        include!(concat!(env!("OUT_DIR"), "/generated.rs"));
-    }
-
-    let generated = generated::generate();
-
-    for (path, _) in generated.into_iter() {
-        let path = if path == "index.html" {
-            // "index.html" redirects to "/"
-            "/".to_owned()
-        } else {
-            "/".to_owned() + path
-        };

-        let (_, status_code) = server.service.get(&path).await;
-        assert_eq!(status_code, 200);
-    }
 }
